actions/terra-k8s-linter/policy/deployments/deny.rego

```diff
@@ -0,0 +1,32 @@
+package main
+
+import data.kubernetes
+
+
+name = input.metadata.name
+
+required_deployment_labels {
+  input.metadata.labels["app.kubernetes.io/name"]
+  input.metadata.labels["app.kubernetes.io/instance"]
+  input.metadata.labels["helm.sh/chart"]
+  input.metadata.labels["app.kubernetes.io/managed-by"]
+}
+
+deny_required_deployment_labels[msg] {
+  input.kind == "Deployment"
+  not required_deployment_labels
+  msg = sprintf("%s must include Kubernetes recommended labels", [name])
+}
+
+deny_replicas[msg] {
+  input.kind == "Deployment"
+  input.metadata.annotations["bio.terra.linter/replicas_exception"] != "enabled"
```

Comment:
I think Justin might've mentioned this and I don't know if it is too late to change now--it isn't a huge deal--the word "exception" is a tiny bit overloaded here, maybe "exemption" conveys what you mean a bit better. Just an idea, don't mean for you to have to go back through all your branches.

Reply:
I can no longer change this, unfortunately. I've already done a chart release.